### ISSUE TITLE
Start recording from homescreen.

### ISSUE THREAD

**Issue by blaueente** [osmandapp/OsmAnd#11195]
# 🚀 feature request

### Description

Homescreen widget to start and stop recording.

### Describe the solution you'd like

There should be a track recording widget one can add to ones home screen. On a single tap, the widget shows some kind of loading indicator, then recording starts widget changes to make recording visible. No additional settings dialog. On another tap, the recording is stopped. Alternatively, it shows a dialog on stopping.
There might be a possibility to configure default options somwhere.
Not needing that concentration to wait for the app loading (on slower phones, or phones with slow SD map storage), or navigate menus might improve both convenience and safety at the same time.

This might also improve the experience a user is complaining about in https://github.com/osmandapp/OsmAnd/issues/10999

### Describe alternatives you've considered

 Have you considered any alternative solutions or workarounds?
"Workaround", workflow as of now: Start Osmand - wait for loading - tap on small track recording icon - get settings dialog - tap on start.

The Osmand icon itself offers a context menu with something like "start recording now". That would be a bit more taps.

An alternative could be to use another app, but then the great integration of track recording into osmand could not be used, such as the track shown on the map, the statistics, etc.

**Comment by vshcherb** [comment]
Yes, it's actually related to our plans for Home screen shortucts.
